### DISCUSSION TITLE
Shipping infrastructure

### DISCUSSION THREAD

**Discussion by dpomt** [OrchardCMS/OrchardCore.Commerce#394] [General] [answered]
Is implementation of shipping (costs) already in progress? 

I saw an issue [https://github.com/OrchardCMS/OrchardCore.Commerce/issues/4](https://github.com/OrchardCMS/OrchardCore.Commerce/issues/4) with some discussion on the topic and asked there.

Any ideas where to integrate a ZIP-based shipping method? Maybe in a class derived from ShoppingCartEventsBase?

Thanks,
Dieter

**Comment by sarahelsaig** [comment, marked as answer]
Please don't open discussions regarding open issues. Thanks.
